cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -4745,9 +4745,8 @@ macro(build_awssdk)
       -DENABLE_TESTING=OFF
       -DENABLE_UNITY_BUILD=ON
       -DOPENSSL_ROOT_DIR=${OPENSSL_ROOT_HINT}
-      "-DCMAKE_INSTALL_PREFIX=${AWSSDK_PREFIX}"
-      "-DCMAKE_PREFIX_PATH=${AWSSDK_PREFIX}")
-
+      -DCMAKE_INSTALL_PREFIX=${AWSSDK_PREFIX}
+      -DCMAKE_PREFIX_PATH=${AWSSDK_PREFIX}\\$<SEMICOLON>${OPENSSL_ROOT_HINT})
```

Comment:
Could you sort this list?

Reply:
Like this? https://github.com/apache/arrow/pull/34215/commits/455e60ad1c973b300557f810cc13eef3092a757c

Reply:
Yes. :-)